--- idevicerestore-4b3e847e1d9a1210049a9e3f1d1caa38650c6617/configure.ac.orig	2026-10-05 16:40:45.000000000 +0200
+++ idevicerestore-4b3e847e1d9a1210049a9e3f1d1caa38650c6617/configure.ac	2026-10-05 20:07:05.299570355 +0200
@@ -83,8 +83,8 @@ case ${host_os} in
   *mingw32*|*cygwin*)
     AC_MSG_RESULT([${host_os}])
     win32=true
-    GLOBAL_CFLAGS+=" -DWIN32 -D__LITTLE_ENDIAN__=1"
-    AC_LDFLAGS+=" -static-libgcc"
+    GLOBAL_CFLAGS="$GLOBAL_CFLAGS -DWIN32 -D__LITTLE_ENDIAN__=1"
+    AC_LDFLAGS="$AC_LDFLAGS -static-libgcc"
     ;;
   darwin*)
     AC_MSG_RESULT([${host_os}])
@@ -105,7 +105,7 @@ if test x$win32 != xtrue; then
 fi
 
 CACHED_CFLAGS="$CFLAGS"
-CFLAGS+=" $libimobiledevice_CFLAGS"
+CFLAGS="$CFLAGS $libimobiledevice_CFLAGS"
 
 # check if libimobiledevice has timeout errors
 AC_CACHE_CHECK(for IDEVICE_E_TIMEOUT in enum idevice_error_t, ac_cv_idevice_error_has_timeout,
